core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/IdeProbeConfig.scala

```diff
@@ -14,4 +15,6 @@ case class IdeProbeConfig(
 
 object IdeProbeConfig extends ConfigFormat {
   implicit val format: ConfigReader[IdeProbeConfig] = exportReader[IdeProbeConfig].instance
+  override implicit def hint[A] = ProductHint[A](ConfigFieldMapping(CamelCase, KebabCase))
```

Comment:
I do not know what to do with this configuration (leave it ? or delete), when we are not using kebab-case in `.conf` file